{"uid":"1887c201c1614ab1","name":"test_category","fullName":"src.tests.ui_tests.test_ui_category#test_category","historyId":"bdd47d824380b8945b6f725c5ea0b675","time":{"start":1743024825360,"stop":1743024825360,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556643222ffa <unknown>\n#1 0x556642ce1970 <unknown>\n#2 0x556642d33385 <unknown>\n#3 0x556642d335b1 <unknown>\n#4 0x556642d823c4 <unknown>\n#5 0x556642d592bd <unknown>\n#6 0x556642d7f70c <unknown>\n#7 0x556642d59063 <unknown>\n#8 0x556642d25328 <unknown>\n#9 0x556642d26491 <unknown>\n#10 0x5566431ea42b <unknown>\n#11 0x5566431ee2ec <unknown>\n#12 0x5566431d1a22 <unknown>\n#13 0x5566431eee64 <unknown>\n#14 0x5566431b5bef <unknown>\n#15 0x556643211558 <unknown>\n#16 0x556643211736 <unknown>\n#17 0x556643221e76 <unknown>\n#18 0x7fd79ba9caa4 <unknown>\n#19 0x7fd79bb29c3c <unknown>","statusTrace":"@pytest.fixture(scope=\"module\")\n    def driver():\n        chrome_options = Options()\n    \n        chrome_options.add_argument(\"--headless\")  # GUI없는 환경에서도 실행 가능\n        chrome_options.add_argument(\"--disable-dev-shm-usage\") #메모리 부족 방지\n        chrome_options.add_argument(\"--no-sandbox\") #샌드박스 비활성화\n        chrome_options.add_argument(\"--disable-gpu\")\n        chrome_options.add_argument(\"--remote-debugging-port=9222\")\n        chrome_options.add_argument(\"--window-size=1920,1080\")\n    \n    \n        service_obj = Service(ChromeDriverManager().install())\n        driver = webdriver.Chrome(service=service_obj,options=chrome_options)\n    \n        driver.get(\"https://www.kurly.com/main\")\n>       WebDriverWait(driver, 10).until(\n            EC.presence_of_element_located((By.XPATH, \"//input[@id='gnb_search']\"))\n        )\n\nsrc/tests/ui_tests/conftest.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"424b26d569c5469b27d62a1e438bce80\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f738efea020>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x556643222ffa <unknown>\nE       #1 0x556642ce1970 <unknown>\nE       #2 0x556642d33385 <unknown>\nE       #3 0x556642d335b1 <unknown>\nE       #4 0x556642d823c4 <unknown>\nE       #5 0x556642d592bd <unknown>\nE       #6 0x556642d7f70c <unknown>\nE       #7 0x556642d59063 <unknown>\nE       #8 0x556642d25328 <unknown>\nE       #9 0x556642d26491 <unknown>\nE       #10 0x5566431ea42b <unknown>\nE       #11 0x5566431ee2ec <unknown>\nE       #12 0x5566431d1a22 <unknown>\nE       #13 0x5566431eee64 <unknown>\nE       #14 0x5566431b5bef <unknown>\nE       #15 0x556643211558 <unknown>\nE       #16 0x556643211736 <unknown>\nE       #17 0x556643221e76 <unknown>\nE       #18 0x7fd79ba9caa4 <unknown>\nE       #19 0x7fd79bb29c3c <unknown>\n\nvenv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1743024825360,"stop":1743024837300,"duration":11940},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556643222ffa <unknown>\n#1 0x556642ce1970 <unknown>\n#2 0x556642d33385 <unknown>\n#3 0x556642d335b1 <unknown>\n#4 0x556642d823c4 <unknown>\n#5 0x556642d592bd <unknown>\n#6 0x556642d7f70c <unknown>\n#7 0x556642d59063 <unknown>\n#8 0x556642d25328 <unknown>\n#9 0x556642d26491 <unknown>\n#10 0x5566431ea42b <unknown>\n#11 0x5566431ee2ec <unknown>\n#12 0x5566431d1a22 <unknown>\n#13 0x5566431eee64 <unknown>\n#14 0x5566431b5bef <unknown>\n#15 0x556643211558 <unknown>\n#16 0x556643211736 <unknown>\n#17 0x556643221e76 <unknown>\n#18 0x7fd79ba9caa4 <unknown>\n#19 0x7fd79bb29c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/src/tests/ui_tests/conftest.py\", line 30, in driver\n    WebDriverWait(driver, 10).until(\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"story","value":"카테고리 기능 테스트"},{"name":"feature","value":"UI 테스트"},{"name":"parentSuite","value":"src.tests.ui_tests"},{"name":"suite","value":"test_ui_category"},{"name":"host","value":"fv-az1315-66"},{"name":"thread","value":"2530-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"src.tests.ui_tests.test_ui_category"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"1887c201c1614ab1.json","parameterValues":[]}